graphql_compiler/macros/macro_edge/validation.py

```diff
@@ -59,11 +64,11 @@ def _validate_macro_ast_with_macro_directives(schema, ast, macro_directives):
                 .format(directive_definition.name, len(macro_data)))
 
 
-def _validate_macro_ast_directives(ast, inside_fold_scope=False):
-    """Check that the macro is using non-macro directives properly.
+def _validate_non_required_macro_definition_directives(ast, inside_fold_scope=False):
+    """Check that the macro is using non-required macro edge definition directives properly.
 
     Restrictions on use of directives:
-    - @output and @output_source are disallowed
+    - @macro_edge, @output and @output_source are disallowed
```

Comment:
I don't think `macro_edge` is necessary here, since per its definition you can't use it outside of a `FIELD_DEFINITION` location (i.e. in schema definitions), otherwise the built-in validation `validate()` call will fail. No need to remove it once it's here, just a bit redundant.

Reply:
Noted.